src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml

```diff
@@ -49,22 +50,40 @@
 
                 <!-- Bulleted list -->
                 <!-- Margin controls the spacing between the text blocks -->
-                <TextBlock TextWrapping="WrapWithOverflow" Margin="0,10,0,0">
-                    <BulletDecorator>
+                <!-- Add a grid so we can make sure the bullets are aligned with the middle
+                     of the first line of text.-->
+                <Grid Margin="0,10,0,0">
+                    <Grid.RowDefinitions>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="20" />
+                        <ColumnDefinition />
+                    </Grid.ColumnDefinitions>
+
+                    <BulletDecorator Grid.Column="0" Grid.Row="0" VerticalAlignment="Center" >
                         <BulletDecorator.Bullet>
                             <Ellipse Height="8" Width="8" Fill="Black" />
                         </BulletDecorator.Bullet>
                     </BulletDecorator>
-                    <Run> check that you have no unsaved files</Run><LineBreak />
-                
-                    <BulletDecorator>
+
+                    <TextBlock Grid.Column="1" Grid.Row="0" TextWrapping="WrapWithOverflow">
+                        <Run> check that you have no unsaved files</Run>
+                    </TextBlock>
+                    
+                    <BulletDecorator Grid.Column="0" Grid.Row="1" VerticalAlignment="Center">
                         <BulletDecorator.Bullet>
                             <Ellipse Height="8" Width="8" Fill="Black" />
                         </BulletDecorator.Bullet>
                     </BulletDecorator>
-                                    
-                    <Run> if the solution is under source control, check that you have no uncommitted changes.</Run>
```

Comment:
Previously the bullets and text were next to each other in the same horizontal stack panel.
That meant that when the text wrapped it appear under the bullet, rather than aligned under the text.

To fix it, I created a grid with three rows and two columns, with the long line of text for the second bullet point spanning two rows.

If I had used a 2 x 2 grid, the alignment of the second bullet point would not have been correct - it would have been centred vertically between the two lines of text.